build_deploy.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+IMAGE="quay.io/cloudservices/content-sources-backend"
```

Comment:
For testing this I used:
```suggestion
IMAGE="${IMAGE:-quay.io/cloudservices/content-sources-backend}"
```

And I passed `IMAGE=quay.io/avisied0/content-sources-backend`

Unsure if any impact, into the environment where it will run; if IMAGE is already defined, it will evoke the image generated is not the expected one.

Reply:
Intended run environment is CI pipelines run by appsre, but I think this change is good to keep it useful outside of that environment without direct modification to the file.